documentation/modules/exploit/multi/http/processmaker_exec.md

```diff
@@ -0,0 +1,93 @@
+## Description
+
+  This module exploits a PHP code execution vulnerability in the default *neoclassic* skin for [ProcessMaker Open Source](http://www.processmaker.com/) which allows any authenticated user to execute PHP code. The vulnerable skin is installed by default in version 2.x prior to 2.5.2 and cannot be removed via the web interface.
+
+
+## Vulnerable Application
+
+  ProcessMaker is an open source, workflow management software suite, which includes tools to automate your workflow, design forms, create documents, assign roles and users, create routing rules, and map an individual process quickly and easily. It's relatively lightweight and doesn't require any kind of installation on the client computer.
+
+  This module has been tested successfully on ProcessMaker Open Source versions 2.5.1, 2.5.0, 2.0.23 on Windows XP SP3; and version 2.0.45 on Debian Linux.
```

Comment:
Lets change this to a bullet or numbered list.

Successfully tested against:

- 2.5.1 on Windows XP SP3
- 2.5.0 on Windows XP SP3
- 2.0.45 on Debian
- 2.0.23 on Windows XP SP3